modules/linear_boltzmann_solvers/lbs_solver/iterative_methods/ags_solver.h

```diff
@@ -48,6 +48,8 @@ class AGSSolver
   double tolerance_;
   bool verbose_;
 
+  double ComputePhiErrorNorm() const;
```

Comment:
Consider changing this to `ComputePhiL2Change` or something like that if you are sticking to using this routine instead of the existing routines.

Reply:
semantics: shouldn't this be ComputeL2ChangePhi? just like it the previous one should be ComputeErrorNormPhi? just wondering.

deeper question to @zhardy-lanl : why this  note" if you are sticking to using this routine instead of the existing routines"

Reply:
Probably `ComputeL2PhiChange` is more consistent IMO. On the deeper question, see my comments on having multiple routines floating around to compute L2 errors/changes. I said that in the event that the routine was abandoned in favor of existing routines to compute L2 norms.

Reply:
Changed to `ComputeL2PhiChange`.